api/v1/controllers/AuthController.js

```diff
@@ -70,8 +80,85 @@ function refreshToken (req, res, next) {
 		});
 }
 
+function requestPasswordReset (req, res, next) {
+	var userEmail = req.query.email;
+
+	UserService
+		.findUserByEmail(userEmail)
+		.then(function (user){
+			return AuthService.generateToken(user, scopes.AUTH);
```

Comment:
I feel like the `UserService` should do this and everything below, right? As in you pass a user to the method `UserService.resetPassword` and then the following is done. It seems like too much business logic is in this controller.